### DISCUSSION TITLE
Please add Gemini 1.5 Flash/Pro and Gemini Flash/Pro Experimental to Vertex AI list of models

### DISCUSSION THREAD

**Discussion by Jasmin68k** [cline/cline#656] [Feature Requests]
Hello!

See title, would be great to have these available:

https://cloud.google.com/vertex-ai/generative-ai/docs/learn/models
https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/gemini-experimental

Thank you!

**Comment by josephomills** [comment]
Exactly. Just like all models on OpenRouter are available in Cline, It would be great to have all Gemini and even other models on Vertex AI added to the list of models. It's still on Claude models at the moment. BTW Gemini 2.0 Flash Exp is also out, so it would be great to have it on there as well.
